README.md

```diff
@@ -1,4 +1,10 @@
-# openfight
+[![CI](https://github.com/humbertodias/openfight/actions/workflows/ci.yml/badge.svg)](https://github.com/humbertodias/openfight/actions/workflows/ci.yml)
```

Comment:
Is it possible to have these map to the username of the git repository instead of having it hard-coded.